src/App.tsx

```diff
@@ -1,27 +1,74 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  // eslint-disable-next-line
-  console.log(timerId);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type Props = {};
+
+type State = {
+  isClockVisible: boolean,
+  clockName: number,
 };
 
+class App extends React.Component<Props, State> {
+  state: State = {
+    isClockVisible: false,
+    clockName: 0,
+  };
+
+  render(): ReactNode {
+    return (
+      <div className="app">
+        <h1 className="app__title">
+          React clock
+        </h1>
+
+        <div className="app__text">
+          Current time:
+        </div>
+
+        <div className="app__clock">
+          {this.state.isClockVisible
+            ? <Clock clockName={this.state.clockName} />
+            : <br />}
```

Comment:
```suggestion
          {this.state.isClockVisible && (<Clock clockName={this.state.clockName} />)}
```
You need to add some styles, use br for extra space bad practice